lib/membrane_rtsp/rtsp.ex

```diff
@@ -131,9 +134,14 @@ defmodule Membrane.RTSP do
     do: request(session, "SET_PARAMETER", headers, body)
 
   @spec teardown(t(), headers()) :: Response.result()
-  @spec teardown(pid()) :: {:error, atom()} | {:ok, Membrane.RTSP.Response.t()}
   def teardown(session, headers \\ []), do: request(session, "TEARDOWN", headers)
 
+  @spec repeat_last_request(t()) :: Response.result()
+  def repeat_last_request(session), do: GenServer.call(session, :repeat_last_request)
```

Comment:
So instead, as we spoke, let's send the response from the TCP decapsulator back to the RTSP session and remove `*_no_response` functions